.github/workflows/metrics.yml

```diff
@@ -36,12 +33,23 @@ jobs:
         if: matrix.backend == 'postgres'
         run: |
           sudo apt-get update
-          sudo apt-get install -y libpq-dev postgresql
-          echo "host    all             all             127.0.0.1/32            md5" > sudo tee -a /etc/postgresql/10/main/pg_hba.conf
-          sudo service postgresql restart && sleep 3
+          sudo DEBIAN_FRONTEND=noninteractive apt-get --purge remove postgresql\* -y
```

Comment:
I did that to create a predictable environment for configuration and testing. Each distro with distinct versions has a different postgresql package installed, and sometimes its default service is up and running (like in github actions containers). By removing all installed postgresql packages, it will stop their services and clusters.

Please consider that the default ubuntu package manager does not ship the latest stable PostgreSQL package (version 16, not even 15).

I chose postgresql version 16 as it is the latest stable version. If you are not happy with the latest stable version, simply changing the `PG_VERSION` job env var can install whatever is suitable for this job.

PS: Previous script tried using the default installed postgresql (currently 14), and then tried to configure postgresql 10 (/etc/postgresql/10/main/pg_hba.conf)! Something that never happened...